reporting.md

```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
 1. [BIRT Reports](reporting_birt.md) for facility level and some MoH HMIS reports 
 2. [Cohort Builder](reporting_cohort_builder.md) for custom data extraction in Excel for more detailed analysis
 
-USING THE COHORT BUILDER
+## USING THE COHORT BUILDER
```

Comment:
@ODORA0 Headings should be title case not capital letters

Reply:
@ssmusoke Done